github/bump_versions/bump_versions.rb

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ruby
+
+# This script bumps the version in all YaST packages
+#
+# You need admin access rights to temporarily disable the GitHub branch protection
+# and allow direct push to "master".
+#
+
+# the GitHub organization
+GH_ORG = "yast".freeze
+
+# install missing gems
+if !File.exist?("./.vendor")
+  puts "Installing the needed Rubygems to ./.vendor/bundle ..."
+  system "bundle install --path .vendor/bundle"
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "octokit"
+
+DRY_RUN = ENV["DRY_RUN"]
+
+# new package version
+NEW_PACKAGE_VERSION = "4.4.0".freeze
+# for checking whether the version has been already bumped
+NEW_PACKAGE_PREFIX = NEW_PACKAGE_VERSION[0..-2]
+
+# some packages use distro based version
+NEW_DISTRO_VERSION = "15.4.0".freeze
```

Comment:
I would group all modificable constant to top, so you do not overlook any when you use script.